lib/models/account.rb

```diff
@@ -5,6 +5,10 @@ def to_s
     string = "#{@name}, #{@uid}, #{@parent_id}, #{@root_id}, #{@time_zone}, #{@sis_id}, #{@workflow}"
   end
 
+  def to_csv
+    row = [@name, @uid, @parent_id, @root_id, @time_zone, @sis_id, @workflow]
```

Comment:
@name, @uid, @parent_uid etc. should be name, uid, parent_uid etc respectively since attr_reader allows those values to be read

Reply:
resolved